projects/cert-manager/cert-manager/manifests/cert-manager.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The cert-manager Authors.
```

Comment:
For posterity: 
```bash
➜  ~ curl -sL https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml | wc -l
    5422
➜  ~ curl -sL https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml | wc -l                  
    5529
```
There are only about 107 lines added to the manifest between 1.9.1 and 1.11.0, but they moved around the CRDs and fields A LOT so the diff looks huge.